src/c2goto/library/kernel/include/linux/gfp.h

```diff
@@ -0,0 +1,67 @@
+
+#define MAX_ALLOC_SIZE 1024*1024 // Maximum allocatable size set to 1MB for example
```

Comment:
Is this a hard requirement? 

If not, then this should be symbolic or obtained through a intrinsic. A TODO is enough for now though